app/presenters/volunteer_stats_presenter.rb

```diff
@@ -0,0 +1,41 @@
+class VolunteerStatsPresenter
+  def initialize(volunteer, time = Time.zone.now)
+    @volunteer = volunteer
+    @time = time
```

Comment:
It doesn't necessarily have to be the current time. Could potentially be a time in the past. I think I'll keep it as `time` for now.